src/haddock/modules/analysis/caprieval/defaults.yaml

```diff
@@ -1,157 +1,186 @@
 reference_fname:
   default: ''
   type: file
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Reference structure
+  short: Structure to be used when calculating the CAPRI metrics.
+  long: Structure to be used when calculating the CAPRI metrics.
+    If none is defined then the lowest scoring structure is selected by default.
+  group: 'analysis'
   explevel: easy
 irmsd:
   default: true
   type: boolean
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Calculate the I-RMSD
+  short: Perform I-RMSD calculations.
+  long: Perform I-RMSD calculations.
+  group: 'analysis'
   explevel: easy
 fnat:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Perform FNAT calculations.
+  long: Perform FNAT calculations.
+  group: 'analysis'
   explevel: easy
 lrmsd:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Perform L-RMSD calculations.
+  long: Perform L-RMSD calculations.
+  group: 'analysis'
   explevel: easy
 ilrmsd:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Perform I-L-RMSD calculations.
+  long: Perform I-L-RMSD calculations.
+  group: 'analysis'
   explevel: easy
 dockq:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Evaluate DockQ.
+  long: Evaluate DockQ.
+  group: 'analysis'
   explevel: easy
 irmsd_cutoff:
   default: 10.0
   type: float
-  min: -9999
+  min: 0.0
   max: 9999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Distance cutoff (Å) to consider the interface
+  short: Distance cutoff (Å) to consider the interface.
+  long: Distance cutoff (Å) to consider the interface.
+  group: 'analysis'
   explevel: easy
 fnat_cutoff:
   default: 5.0
   type: float
-  min: -9999
+  min: 0.0
   max: 9999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Distance cutoff (Å) to consider the interface
+  short: Distance cutoff (Å) to consider the interface.
+  long: Distance cutoff (Å) to consider the interface.
```

Comment:
"to consider the interface" is rather poorly defined.

May-be better: used to define interface contacts

And same comment for the RMSD values, but in that case rather "to define interface residues"

Reply:
updated